### ISSUE TITLE
image_demo.py does not support  show mask

### ISSUE THREAD

**Issue by ioir123ju** [open-mmlab/mmdetection#2812]
**Checklist**
config/ms_rcnn_x101_64x4d_fpn_1x_coco.py

**Describe the bug**
image_demo.py's result is Binary mask.
But show_result function need RLE.
So it's wrong

**Reproduction**
1. What command or script did you run?
```
 python demo/image_demo.py  configs/ms_rcnn/my_ms_rcnn_x101_64x4d_fpn_1x_coco.py work_dirs/my_ms_rcnn_x101^C4x4d_fpn_1x_coco/latest.pth
```

**Error traceback**
```
File "demo/image_demo.py", line 105, in main
    show_result_pyplot(model, img, result, score_thr=args.score_thr)
  File "/home/juzheng/code/competition/mmdetection/mmdet/apis/inference.py", line 146, in show_result_pyplot
    img = model.show_result(img, result, score_thr=score_thr, show=False)
  File "/home/juzheng/code/competition/mmdetection/mmdet/models/detectors/base.py", line 214, in show_result
    mask = maskUtils.decode(segms[i]).astype(np.bool)
  File "/home/juzheng/anaconda3/envs/python3.6/lib/python3.6/site-packages/pycocotools/mask.py", line 91, in decode
    return _mask.decode([rleObjs])[:,:,0]
  File "pycocotools/_mask.pyx", line 146, in pycocotools._mask.decode
  File "pycocotools/_mask.pyx", line 128, in pycocotools._mask._frString

```

**Comment by hellock** [comment]
Should be fixed in #2786

**Comment by hellock** [comment]
Fixed in #2824.
